### DISCUSSION TITLE
I want to change subrow css

### DISCUSSION THREAD

**Discussion by ChanoPark** [KevinVandy/material-react-table#261] [Q&A] [answered]
Hello!

I refer to the following page to implement subrow.
https://www.material-react-table.com/docs/guides/expanding-sub-rows

As mentioned in the previous question, the size of the table is fixed, so there is a problem that the indentation of subrows does not occur. So I'm trying to change the font-weight to put a difference.

So when I declared the columns I wrote:
![image](https://user-images.githubusercontent.com/77626299/208029938-ba9e8967-8b78-48ba-9ec0-93c30971de87.png)

If `data.data.select.sub` is true, isExpand set true. 
`row.depth` is 0, I decide subrow and change the font-weight to `bold`. otherwise, `normal`.

Contrary to expectations, a random line turns bold every time it renders. (Presumed to be order dependent)

Can I change the color and thickness of only the row except the sub row?

Thank you!

**Comment by KevinVandy** [comment, marked as answer]
Welp don't do it that way. Manual mutations like that in React are not good. Just use the `muiTableBodyCellProps` directly in your column definitions. I suggest reading the [Customizing Components Guide Docs](https://www.material-react-table.com/docs/guides/customize-components) where this is discussed

You probably want something like

```js
muiTableBodyCellProps: ({ cell, row }) => ({ //get your data from row.original probably
  sx: {
    fontWeight: condition ? 'bold' : 'normal'
  }
})
```

**Reply by ChanoPark** [reply]
Thank you for always :)
